### DISCUSSION TITLE
Markmap integration!

### DISCUSSION THREAD

**Discussion by magnussentio** [hakimel/reveal.js#3436] [Ideas]
Markmap.js is the cleanest mindmapping javascript library! Would love for an integration to be made (even though it can be used without one!)

**Comment by t-fritsch** [comment]
Hey @magnussentio thank you for your suggestion, anyway I think this kind of integration would better fit in a plugin than being part of core.

If markmap.js is a tool you know well, you could maybe try creating a plugin to ease its integration ? Did you have a chance to look a the documentation here : https://revealjs.com/creating-plugins/ ?
